tardis/io/tests/data/csvy_test_config.yml

```diff
@@ -0,0 +1,77 @@
+# Example YAML configuration for TARDIS
+tardis_config_version: v1.0
+
+supernova:
+  luminosity_requested: 8.770 log_lsun
+  time_explosion: 16 day
+  distance: 8.32 Mpc
+
+# standard atomic data base; get it from the tardis-refdata repository
+atom_data: /home/mew488/Research/TARDIS/tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5
```

Comment:
Wouldn't it need to reference the tardis repo itself? As in: github.com/tardis-sn/tardis-refdata/blob/master/atom_data/kurucz_cd23_chianti_H_He.h5

Reply:
Yes good point. Although this file was used for local testing and is not necessary for the pytests.

Reply:
@Youssef15015 that won't work as tardis can not fetch online data. @marxwillia you can just use `kurucz_...` - TARDIS can download the necessary data.